test/test_read_csv.py

```diff
@@ -0,0 +1,9 @@
+from src.read_csv import readFile

```

Comment:
can you format your file and make sure you follow the python style guide here: https://www.python.org/dev/peps/pep-0008/